### PR TITLE
feat: Update the design for label management page

### PR DESCRIPTION
**Pull request by pranavrajs**
This PR is part of the settings design update series. It updates the design for the label management page. I've made a few changes to the SettingsLayout page to reduce boilerplate code.

| Light | Dark |
| -- | -- |
| Screenshot 2024-08-12 at 8 33 31 AM | Screenshot 2024-08-12 at 8 33 43 AM |

### CODE REVIEW

File: app/javascript/dashboard/i18n/locale/en/labelsMgmt.json [hunk 1]
```diff
@@ -3,13 +3,19 @@
     "HEADER": "Labels",
     "HEADER_BTN_TXT": "Add label",
     "LOADING": "Fetching labels",
+    "DESCRIPTION": "Labels help you categorize and prioritize conversations and leads. You can assign a label to a conversation or contact using the side panel.",
+    "LEARN_MORE": "Learn more about labels",
     "SEARCH_404": "There are no items matching this query",
     "SIDEBAR_TXT": "<p><b>Labels</b> <p>Labels help you to categorize conversations and prioritize them. You can assign label to a conversation from the sidepanel. <br /><br />Labels are tied to the account and can be used to create custom workflows in your organization. You can assign custom color to a label, it makes it easier to identify the label. You will be able to display the label on the sidebar to filter the conversations easily.</p>",
```
Comment (iamsivin) [inline app/javascript/dashboard/i18n/locale/en/labelsMgmt.json:9]: @pranavrajs Do we need these translations?

File: app/javascript/dashboard/routes/dashboard/settings/labels/Index.vue [hunk 2]
```diff
@@ -1,138 +1,137 @@
...
-                  <span
-                    class="label-color--display"
-                    :style="{ backgroundColor: label.color }"
-                  />
-                  {{ label.color }}
-                </div>
-              </td>
-              <td class="button-wrapper">
+          <tr v-for="(label, index) in records" :key="label.title">
+            <td class="py-4 pr-4">
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/labels/Index.vue:116]: @pranavrajs Could we update the padding to also support RTL view?
```suggestion
            <td class="py-4 ltr:pr-4 rtl:pl-4">
```

**Review by iamsivin — COMMENTED** [review]
@pranavrajs LGTM. Please review the inline comments.

File: app/javascript/dashboard/i18n/locale/en/labelsMgmt.json (hunk 1 above)
Comment (pranavrajs) [reply app/javascript/dashboard/i18n/locale/en/labelsMgmt.json:9]: Good one! Let me remove it.

File: app/javascript/dashboard/routes/dashboard/settings/labels/Index.vue (hunk 2 above)
Comment (pranavrajs) [reply app/javascript/dashboard/routes/dashboard/settings/labels/Index.vue:116]: Makes sense, let me make the change.

Comment (pranavrajs) [reply app/javascript/dashboard/routes/dashboard/settings/labels/Index.vue]: Done.

File: app/javascript/dashboard/i18n/locale/en/labelsMgmt.json (hunk 1 above)
Comment (pranavrajs) [reply app/javascript/dashboard/i18n/locale/en/labelsMgmt.json]: Done.

**Comment by pranavrajs** [comment]
@iamsivin Made the changes.

**Review by iamsivin — APPROVED** [review]

**Pull request merged**